### ISSUE TITLE
bug(*): fix mobile view.

### ISSUE THREAD

**Issue by GabuTheDev** [cyperdark/osuck-skins#16]
### browser

Brave

### bug description

barmenu centering on... what?! clearly not on viewport size.

https://github.com/cyperdark/osuck-skins/assets/107962621/01c2e17f-464b-4172-b78c-aa3de8f312a6

scroll randomly stops?! scroll stutter? 

https://github.com/cyperdark/osuck-skins/assets/107962621/299469c4-a79e-4fd2-b35a-42b2915bd0b1

here's a more obvious video from a skin page:

https://github.com/cyperdark/osuck-skins/assets/107962621/cba1e7e4-a58a-4d67-9459-8b8d49946356

also, skin properties

 (HD, SD, ANIMATED) are being covered by the quick link button? and they are not horizontally scrollable.

![photo_2023-10-12_17-24-04](https://github.com/cyperdark/osuck-skins/assets/107962621/b72ed60f-8191-451a-b50d-80efb120cad8)

and image view is... weirdly not scaling to the screen width... again viewport issue.

https://github.com/cyperdark/osuck-skins/assets/107962621/af0b9443-8279-4e31-b8c5-bd08e8ceb2bb

**Comment by cyperdark** [comment]
Check image view and bar menu in current version

**Comment by GabuTheDev** [comment]
![Screenshot_2023-12-14-22-22-32-37_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/cyperdark/osuck-skins/assets/107962621/aea3a622-6c37-4b36-a1eb-92fca7b2d261)

![Screenshot_2023-12-14-22-22-45-98_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/cyperdark/osuck-skins/assets/107962621/dbb9ac34-28df-45f5-b729-baa695d149cf)

The navbar is still not fitted into the viewport

**Comment by cyperdark** [comment]
it's because there is too much things in navbar, but overall better than before?

**Comment by GabuTheDev** [comment]
> it's because there is too much things in navbar, but overall better than before?

Yeah the overall viewport is back to normal. Just the navbar being weird

**Comment by cyperdark** [comment]
about weird scroll shutter, does your phone have any scroll smoothing things?

**Comment by GabuTheDev** [comment]
> about weird scroll shutter, does your phone have any scroll smoothing things?
> 

My os has this kind of.. scroll end thing... Idk how to describe it:
https://github.com/cyperdark/osuck-skins/assets/107962621/97172884-db10-4a77-a8db-17389965e82a


But apparently the scroll shutter is gone!?

**Comment by cyperdark** [comment]
huh, nice but how

**Comment by GabuTheDev** [comment]
> huh, nice but how

I don't know? it does not happen anymore? probably was a browser issue? which I wouldn't really rely on because I've checked the issues on multiple browsers & devices.

**Comment by GabuTheDev** [comment]
![image](https://github.com/cyperdark/osuck-skins/assets/107962621/2221b978-850f-4a5e-ba9f-069d53f46501)
Another one :D
